examples/CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ function(add_example name)
 endfunction()
 
 if(UNIX)
+    add_definitions(-DZENOH_LINUX)
```

Comment:
If you define it here as well, then I suggest that you use instead the `CMAKE_SYSTEM_NAME` in a similar way to the root `CMakeLists.txt`.